### PR TITLE
Switching out a video for the intro gif

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site link](https://review.docs.microsoft.com/en-us/office/dev/scripts/overview/excel?branch=AlexJ-Video)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8dc92e2](https://github.com/OfficeDev/office-scripts-docs/commits/8dc92e2710ae1455ae0899dc40716b48da62cae0)_: 

:clock10: **Pending**: waiting for processors ([6](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 6 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8dc92e2](https://github.com/OfficeDev/office-scripts-docs/commits/8dc92e2710ae1455ae0899dc40716b48da62cae0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Video/docs/overview/excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C24%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002242144344491-90%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2B3ejlsJhOl1Yjvbc8uGh6w15wdSpo21erNk%2FsVpeyrk%3D&st=2020-02-24T21%3A40%3A45Z&se=2020-03-26T21%3A45%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/excel.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 ---
 
 # Office Scripts in Excel on the web (preview)
 
 Office Scripts in Excel on the web let you automate your day-to-day tasks. You can record your Excel actions with the Action Recorder, which creates a script. You can also create and edit scripts with the Code Editor. This series of documents teaches you how to use these tools. You'll be introduced to the Action Recorder and see how to record your frequent Excel actions. You'll also learn how to make or update your own scripts with the Code Editor.
 
-![A gif showing a script being recorded and replayed on different worksheets](../images/teaser-animation.gif)
+<br>
+
+> [!VIDEO https://www.microsoft.com/videoplayer/embed/RE4qdFF]
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/excel.md]: can you include alt text?

Comment (ElizabethSamuel-MSFT) [inline docs/overview/excel.md]: also I don't think you need the >

Comment (AlexJerabek) [reply docs/overview/excel.md]: I'll try it out.

Comment (AlexJerabek) [reply docs/overview/excel.md]: I'm following [this reference page](https://review.docs.microsoft.com/en-us/office-authoring-guide/add-a-video-to-article?branch=master), which seems to indicate I do. I'll test.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[36cfed4](https://github.com/OfficeDev/office-scripts-docs/commits/36cfed4d918d5e2989f4d25538f45c71c0398d4c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Video/docs/overview/excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C24%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002242220467295-90%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=EZhKNTtH7XwsTwhnMktZRc45LTYRRF2XTcsohIAJKDU%3D&st=2020-02-24T22%3A17%3A10Z&se=2020-03-26T22%3A22%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b673e78](https://github.com/OfficeDev/office-scripts-docs/commits/b673e78422e25ce5c76f057b42091d36e13118c5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/excel.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-Video/docs/overview/excel.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C24%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202002242235275989-90%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ujfZ2aFBhc1JfSRE4Q8xhfMHHy2c4mU4RSOFtbw%2B9I4%3D&st=2020-02-24T22%3A31%3A55Z&se=2020-03-26T22%3A36%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**